contributing-guides/maintainers-guide.md

```diff
@@ -89,6 +89,8 @@ as a guideline for current and future maintainers.
   use the `Squash and merge` method.
   Don't forget to clean up the body of the squashed commit message, especially the `Co-authored by:` part.
   (You can always use the body if you want to include useful information.)
+  
+ Note: For some PRs with reviews and proofreading from contributors or native speakers outside of the tldr-pages organization the `Co-authored by` part must be included in the merge commit message.
```

Comment:
```suggestion
 Note: For some PRs with reviews and proofreading from contributors or native speakers outside of the tldr-pages organization the `Co-authored by` part must be included in the merge commit message.  
```
Adding 2 spaces to make [`Line 94`](https://github.com/tldr-pages/tldr/blob/fee7444ea1349418f91b9fe7cb388b63418fa60c/contributing-guides/maintainers-guide.md) go to next line

Reply:
done ;)